.up/examples/cluster-claim.yaml

```diff
@@ -6,17 +6,14 @@ metadata:
   annotations:
     uptest.upbound.io/pre-delete-hook: testhooks/delete-release.sh
 spec:
-  id: platform-ref-aws
   parameters:
-    version: "1.26"
+    id: platform-ref-aws
+    region: us-west-2
+    version: "1.27"
+    iam:
+      roleArn: arn:aws:iam::609897127049:role/AWSReservedSSO_AdministratorAccess_d703c73ed340fde7
```

Comment:
is it intentional account/role exposure?

Reply:
AWS Auth Configmap ist part of Composition - so one Role to grant access and Demo the AWS Auth Configmap